### DISCUSSION TITLE
Is there any way to deploy directus in serverless environment?

### DISCUSSION THREAD

**Discussion by niwasmala** [directus/directus#5912] [Archive]
Hello all. 

Is there any way to deploy directus in serverless environment? For example splitting the deployment between API & frontend admin. The API deployed to AWS Lambda, and the frontend admin to AWS Amplify. I think that would be a great help since AWS Lambda & Amplify has free tier usage and it helps for small startup to start without worrying about cost.

I don't find any documentation on how to do this. Will try this week. 

Thanks.

**Comment by rijkvanzanten** [comment]
Technically speaking, the express app that powers the API is exported separately, so you could wrap that in something like serverless-express to run it under lambda. That being said, we haven't done any testing with this, and don't _officially_ support it at this time.

Ref https://github.com/directus/directus/discussions/5309, https://github.com/directus/directus/discussions/4010
